.github/workflows/tests.yml

```diff
@@ -31,11 +31,11 @@ jobs:
       run: echo "IMAGE_NAME=meilisearch_digitalocean_ci_test_$(date +'%d-%m-%Y-%H-%M-%S')" >> $GITHUB_ENV
     - name: Build image
       run: python3 tools/build_image.py $IMAGE_NAME --no-analytics
-    - name: Test image
-      run: python3 tools/test_image.py $IMAGE_NAME
-    - name: Clean image
-      if: ${{ always() }}
-      run: python3 tools/destroy_image.py $IMAGE_NAME
+    # - name: Test image
+    #   run: python3 tools/test_image.py $IMAGE_NAME
+    # - name: Clean image
+    #   if: ${{ always() }}
+    #   run: python3 tools/destroy_image.py $IMAGE_NAME
```

Comment:
Do you want to leave the comments here?

Reply:
No, the Ci is broken, I'm doing some tests ... sorry for the request before I saw the error